src/server/superPhramOrders.ts

```diff
@@ -0,0 +1,170 @@
+import * as fetch from "node-fetch";
+import { DataControl } from "@remult/angular";
+import { DateOnlyField, Entity, Field, IdEntity, ProgressListener, Remult} from "remult";
+import { Roles } from "../app/users/roles";
+import { LogisticsLog } from "../app/invoices-sent-to-logistics/logistics-log";
+import { DateOnlyValueConverter } from "remult/valueConverters";
+import { sendDataToFtp } from "../app/invoices-sent-to-logistics/sendFtp";
+import { LogisticsController } from "../app/invoices-sent-to-logistics/logistics-controller";
+import { callRivhit } from "../app/create-invoice/invoice";
+
+export async function callSuperPharm(api: string) {
+  const { SUPERPHARMSHOPKEY, SUPERPHARMSHOPURL } = process.env;
+
+  let headers = new fetch.Headers();
+  headers = {
+    Accept: "application/json",
+    Authorization: SUPERPHARMSHOPKEY,
+  };
+
+  let r = await fetch.default(SUPERPHARMSHOPURL + "/api/orders", {
+      method: "GET",
+      headers,
+    })
+    .then((response) => { return response.json()})
+    .catch((error) => {
+      throw error?.client_message || error;
+    });
+  return r.orders;
+}
+
+@Entity<OrdersSentToRivhit>("OrdersSentToRivhit", {
+  allowApiRead: Roles.admin,
+  defaultOrderBy: (self) => [
+    self.invoiceDate.descending(),
+    self.transmitDate.descending(),
+  ],
+})
+export class OrdersSentToRivhit extends IdEntity {
```

Comment:
The name of the entity conflicts with the existing entity called "OrdersSentToRivhit"

You should fix that in the class name and in the key sent as the first parameter to the @Entity decorator

Reply:
What about this?